pyproject.toml

```diff
@@ -1,19 +1,51 @@
 [tool.flake8]
-filename = ["./elements/**/*.py", "./graders/**/*.py", "./python/*.py"]
+filename = [
+    "./elements/**/*.py",
+    "./graders/**/*.py",
+    "./python/*.py",
+    "./exampleCourse/**/*.py",
+    "./testCourse/**/*.py"
+]
+exclude = [
+    # These files are intentionally not parseable as Python.
+    "./exampleCourse/questions/demo/autograder/codeUpload/submitted_code_samples/fib_error_binary_file/fib.py",
+    "./exampleCourse/questions/demo/autograder/codeUpload/submitted_code_samples/fib_error_large_text_file/fib.py",
+]
+per-file-ignores = [
+    # Files related to the Python autograder will often intentionally appear
+    # broken in isolation. We'll allow specific errors in these files to
+    # account for that.
+    "exampleCourse/questions/**/tests/setup_code.py: F401, F821",
```

Comment:
Looking again, does `setup_code` need to have F821? That's an error for undefined variable names, but the `setup_code.py` should be setting all variable names that get used later (I think).

Reply:
After looking, realized that `setup_code.py` can use stuff from the `data` dict.